decidim-elections/app/cells/decidim/elections/election_m_cell.rb

```diff
@@ -91,6 +91,23 @@ def footer_button_text
       def statuses
         []
       end
+
+      def needs_to_show_remaining_time?
+        model.end_time > Time.now.utc && model.end_time < 12.hours.from_now.utc
```

Comment:
I would change the `Time.now` to `Time.current` as the later gets the time from the rails app environment (UTC)

```suggestion
        model.end_time > Time.current && model.end_time < 12.hours.from_now.utc
```

👀 https://discuss.rubyonrails.org/t/time-now-vs-time-current-vs-datetime-now/75183/2